README.md

```diff
@@ -225,16 +225,26 @@ You can of course browse the source on GitHub & in the source tree you download,
 
 Zulip Terminal uses [urwid](http://urwid.org/) to render the UI components in terminal. Urwid is an awesome library through which you can render a decent terminal UI just using python. [Urwid's Tutorial](http://urwid.org/tutorial/index.html) is a great place to start for new contributors.
 
+### Getting Zulip Terminal code and connecting it to upstream
+
+First, fork the `zulip/zulip-terminal` repository ([see how](https://docs.github.com/en/get-started/quickstart/fork-a-repo)) and then clone your forked repository locally (the following will place the repository in the current directory):
+```
+$ git clone --config pull.rebase git@github.com:YOUR_USERNAME/zulip-terminal.git
```

Comment:
Does other documentation like this describe `YOUR_USERNAME`?

Reply:
Only [Zulip](https://zulip.readthedocs.io/en/latest/git/cloning.html#step-1b-clone-to-your-machine) has it as `YOURUSERNAME`. Zulip-mobile and desktop don't have instructions regarding cloning the _forked_ repository.